package.json

```diff
@@ -8,26 +8,26 @@
     "eject": "expo eject"
   },
   "dependencies": {
-    "expo": "^36.0.0",
-    "expo-asset": "~8.0.0",
-    "expo-font": "~8.0.0",
-    "firebase": "6.6.0",
+    "expo": "^37.0.0",
+    "expo-asset": "~8.1.3",
+    "expo-font": "~8.1.0",
+    "firebase": "7.9.0",
     "formik": "2.1.4",
     "react": "16.9.0",
     "react-dom": "16.9.0",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-36.0.1.tar.gz",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.0.tar.gz",
```

Comment:
I think it's better to use `37.0.1` 😄 ([see this](https://github.com/expo/react-native/releases))

Reply:
Ah, it slipped me. I didn't know about it. Also, creating a new expo project support the patch versions? I thought it only used latest major/minor released SDK versions.